src/api/app/models/bs_request_action_maintenance_incident.rb

```diff
@@ -39,6 +39,7 @@ def get_releaseproject(pkg, tprj)
       end
       releaseproject = tprj
     end
+    raise RemoteProjectError.new "A release project have to be a local project" unless releaseproject.kind_of? Project
```

Comment:
I like it because it would make multiple lines way more compact.

I dis-like it because we would if the same if condition in two lines now. And there is the risk that it will divert on future code changes.

So, I would like to hear a third opinion about this :)